x-pack/platform/plugins/shared/search_connectors/server/lib/indices/text_analysis.ts

```diff
@@ -0,0 +1,295 @@
+/*
```

Comment:
I don't think we want this used in 9.X if you're not positive its used please remove it. if it is used we should figure out where and remove that IMO

Reply:
it is used by this code https://github.com/elastic/kibana/blob/b9264e57f92a2b9ce1e25ce3f791282b36a92337/x-pack/platform/plugins/shared/search_connectors/server/lib/indices/create_index.ts#L74